pkgs/games/teeworlds/default.nix

```
@@ -21,6 +21,11 @@ stdenv.mkDerivation rec {
     # Can't use fetchpatch or fetchpatch2 because of https://github.com/NixOS/nixpkgs/issues/32084
     # Using fetchurl instead is also not a good idea, see https://github.com/NixOS/nixpkgs/issues/32084#issuecomment-727223713
     ./rename-VERSION-to-VERSION.txt.patch
+    (fetchpatch {
+      name = "CVE-2021-43518.patch";
+      url = "https://salsa.debian.org/games-team/teeworlds/-/raw/a6c4b23c1ce73466e6d89bccbede70e61e8c9cba/debian/patches/CVE-2021-43518.patch";
```

```suggestion
      url = "https://github.com/teeworlds/teeworlds/commit/91e5492d4c210f82f1ca6b43a73417fef5463368.patch";
```
Why fetch the patch from a third party when you can fetch it directly from the original source? The only real difference is the use of `minimum()` vs `min()`. Both functions exist and seem to work identically. Teeworlds changed all uses of `min()` to `minimum()` in https://github.com/teeworlds/teeworlds/pull/2924 for no real reason other than a smaller diff to ddnet. No imports or function definitions were changed, so I think using either one is fine.

---

`minimum()` is not defined in Teeworlds 0.7.5. The PR you mention is only present in  `master` so we cannot pull https://github.com/teeworlds/teeworlds/commit/7e81ae1aac5dc4dc80b064262a4cef57d1361ce4 directly without making some changes. Given those changes has already been made by the Debian maintainers I preferred to pull that instead of maintaining our own patch.

---

You're right, sorry. I thought I tested compilation with this change but it wasn't done compiling yet 😅

---

Ah, I see now where the definition was changed in the code. Sorry about that.